include/fmt/chrono.h

```diff
@@ -1627,6 +1632,15 @@ class tm_writer {
     out_ = copy_str<Char>(std::begin(buf), std::end(buf), out_);
   }
 
+  void on_am_pm_lower() {
+    if (is_classic_) {
+      *out_++ = tm_hour() < 12 ? 'a' : 'p';
+      *out_++ = 'm';
+    } else {
+      format_localized('P');
```

Comment:
Please add a test case to ensure that it actually works with localized formatting.

Reply:
I don’t know how to do this. I wrote the code by blindly copying and modifying, not by deeply understanding what’s going on. 🙂

Reply:
Something along the lines of https://github.com/fmtlib/fmt/blob/050293646fc0337cfdb5084bf16f3d6efc84e9d8/test/chrono-test.cc#L756-L757
but with `%P` instead of `%a` and in a separate test case.

Reply:
Any updates? We'll need to have locale path tested before this can be merged.